src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -1197,6 +1152,14 @@ def main_v03(argv):
             showtiming,
             verbose,
             debuglevel,
+            run_results,
+            data_assimilation_parameters.get("wrf_hydro_lastobs_file",False),
```

Comment:
just pass the whole data_assimilation_parameters dictionary.

Reply:
I don't mind doing so, but I am usually asked not to send everything as it is just more information being passed. I'm not sure if it will ever have optimization consequences.

Reply:
Removing in an upcoming commit.